app/models/embed/was_seed_thumbs.rb

```diff
@@ -0,0 +1,46 @@
+module Embed
+  class WasSeedThumbs
+    attr_reader :druid
+    def initialize(druid)
+      @druid = druid
+    end
+
+    def get_thumbs_list
+      raise ResourceNotAvailable if response.nil?
+      
+      @thumbs_list ||= begin 
+        result = JSON.parse(response)
+        result['thumbnails']
+      end
+    end
+    
+    def response
+      @response ||= begin
+        conn = Faraday.new(url: was_thumbs_url)
+        response = conn.get do |request|
+          request.options = {
+            timeout: 2,
+            open_timeout: 2
+          }
+          request.params['format']='json'
+        end
+        raise ResourceNotAvailable unless response.success?
+        response.body
+      rescue Faraday::Error::ConnectionFailed => error
+        nil
+      rescue Faraday::Error::TimeoutError => error
+        nil
+      end
+    end     
+    
+    def was_thumbs_url
+      "#{Settings.was_thumbs_url}/#{@druid}"
+    end
+    
+    class ResourceNotAvailable < StandardError
```

Comment:
That's what I thought when copying the error. I will add another issue to make this outside the development of the was-seed as I will touch non-webarchive-seed files.